### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by rgraber**
Add more explicit testing instructions in README

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -29,6 +29,10 @@ One Time Setup
 
   # Set up a virtualenv using virtualenvwrapper with the same name as the repo and activate it
   mkvirtualenv -p python3.8 edx-arch-experiments
+  
+Local testing
+~~~~~~~~~~~~~
+In an lms or cms shell, ``pip install git+https://github.com/edx/edx-arch-experiments.git@mybranch``. Make sure to uninstall and reinstall after every change to your local branch.
```
Comment (dianakhuang) [inline README.rst:35]: I think `pip install -e` is a better option here?

Comment (robrap) [reply README.rst:35]: I know about the `/src` directory in devstack that could be used, but not sure if we need instructions for both devstack and Tutor regarding where the code should live such that it is available to the shell?

**Review by robrap — APPROVED** [review]
Looks good. Consider adding a note to local testing section, or somewhere, that the plug-in config will automatically be picked up once installed. If already in the doc, ignore this comment.

**Comment by robrap** [comment]
Also, are there any GitHub workflows we need? I imagine we should have quality running, as an example.

**Pull request merged**